ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -146,11 +146,12 @@ object JdbcIndexer {
       for {
         // Note: the LedgerDao interface is only used for initialization here, it can be released immediately
         // after initialization is finished. Hence the use of ResourceOwner.use().
+        // TODO append-only: Rewrite the initialization using the StorageBackend interface, and remove the use of LedgerDao from this class
```

Comment:
this I accidentally already did in another PR, you can as good as refrain changing this file (one less conflict to worry about)